pkg/cmd/crud/crud.go

```diff
@@ -109,6 +111,11 @@ func generateCrudFiles() error {
 // generateFileFromTemplate uses the passed contents and data object of a
 // template to generate a new file using the specified file name and output path
 func generateFileFromTemplate(name, path, tmplContent string, tmplData interface{}) error {
+	// Sets appropriate target path for the locale file
+	if name == "crud.en.yaml" {
+		path = "./cmd/abc/locales/en"
```

Comment:
I'd prefer this PR to be not merged right now, since the path has been hardcoded for now.
Once the `charmil init` command has been completely set up, the following change can be made to make the path dynamic:

```suggestion
		path = "./cmd/[NAME_OF_CLI]/locales/en"
```

Reply:
charmil init is working, just a small issue. 
I have used a templateContext struct in which I have asked github owner, repo, cliname. So I think we can save cliname somewhere to take reference.

you can walk the directory, cmd/CLI_NAME. So you can get the cliname from here right?

Reply:
> charmil init is working, just a small issue with .git, not working for already initialized git folders.

Yes, that's not the issue.
I was hoping we could save the CLI metadata from `charmil init` somewhere.

 > I have used a templateContext struct in which I have asked github owner, repo, cliname. So I think we can save cliname somewhere to take reference.

This is exactly what I was talking about. I'll have a look at the `templateContext`. 

Thanks!